files/KoreBuild/scripts/commands/Install-Tools.ps1

```diff
@@ -0,0 +1,106 @@
+#requires -version 4
+
+Set-StrictMode -Version 2
+
+<#
+.SYNOPSIS
+Installs tools if required.
+
+.PARAMETER ToolsSource
+The base url where build tools can be downloaded.
+
+.PARAMETER DotNetHome
+The directory where tools will be stored on the local machine.
+#>
+function Install-Tools(
+    [Parameter(Mandatory = $true)]
+    [string]$ToolsSource,
+    [Parameter(Mandatory = $true)]
+    [string]$DotNetHome) {
+
+    $ErrorActionPreference = 'Stop'
```

Comment:
Hopefully I understood the intent and functions well enough to split this off properly here.